ci/travis/determine_tests_to_run.py

```diff
@@ -0,0 +1,112 @@
+                               stderr=err.decode()))
+
+    return list(filter(None, (s.strip() for s in out.decode().splitlines())))
+
+
+if __name__ == "__main__":
+
+    RAY_CI_TUNE_AFFECTED = "0"
+    RAY_CI_RLLIB_AFFECTED = "0"
+    RAY_CI_JAVA_AFFECTED = "0"
+    RAY_CI_PYTHON_AFFECTED = "0"
+    RAY_CI_LINUX_WHEELS_AFFECTED = "0"
+    RAY_CI_MACOS_WHEELS_AFFECTED = "0"
+
+    if os.environ["TRAVIS_EVENT_TYPE"] == "pull_request":
+
+        files = list_changed_files(os.environ["TRAVIS_COMMIT_RANGE"].replace(
+            "...", ".."))
+
+        for changed_file in files:
+            if changed_file.startswith("python/ray/tune/"):
+                RAY_CI_TUNE_AFFECTED = "1"
+                RAY_CI_RLLIB_AFFECTED = "1"
+                RAY_CI_LINUX_WHEELS_AFFECTED = "1"
+                RAY_CI_MACOS_WHEELS_AFFECTED = "1"
+            elif changed_file.startswith("python/ray/rllib/"):
+                RAY_CI_RLLIB_AFFECTED = "1"
+                RAY_CI_LINUX_WHEELS_AFFECTED = "1"
+                RAY_CI_MACOS_WHEELS_AFFECTED = "1"
+            elif changed_file.startswith("python/"):
+                RAY_CI_TUNE_AFFECTED = "1"
+                RAY_CI_RLLIB_AFFECTED = "1"
+                RAY_CI_PYTHON_AFFECTED = "1"
+                RAY_CI_LINUX_WHEELS_AFFECTED = "1"
+                RAY_CI_MACOS_WHEELS_AFFECTED = "1"
+            elif changed_file.startswith(
+                    "java/") and not changed_file.startswith("java/doc/"):
+                RAY_CI_JAVA_AFFECTED = "1"
+            elif changed_file.startswith("doc/"):
+                # nothing is run but linting in this case
+                pass
+            elif changed_file.startswith("examples/"):
+                pass
+            elif changed_file.startswith("dev/"):
+                pass
+            elif changed_file.startswith("docker/"):
+                pass
+            elif changed_file.startswith("kubernetes/"):
+                pass
+            elif changed_file.startswith("site/"):
+                pass
+            elif changed_file.startswith("src/"):
+                RAY_CI_TUNE_AFFECTED = "1"
+                RAY_CI_RLLIB_AFFECTED = "1"
+                RAY_CI_JAVA_AFFECTED = "1"
+                RAY_CI_PYTHON_AFFECTED = "1"
+                RAY_CI_LINUX_WHEELS_AFFECTED = "1"
+                RAY_CI_MACOS_WHEELS_AFFECTED = "1"
+            else:
+                RAY_CI_TUNE_AFFECTED = "1"
+                RAY_CI_RLLIB_AFFECTED = "1"
+                RAY_CI_JAVA_AFFECTED = "1"
+                RAY_CI_PYTHON_AFFECTED = "1"
+                RAY_CI_LINUX_WHEELS_AFFECTED = "1"
+                RAY_CI_MACOS_WHEELS_AFFECTED = "1"
+    else:
+        RAY_CI_TUNE_AFFECTED = "1"
+        RAY_CI_RLLIB_AFFECTED = "1"
+        RAY_CI_JAVA_AFFECTED = "1"
+        RAY_CI_PYTHON_AFFECTED = "1"
+        RAY_CI_LINUX_WHEELS_AFFECTED = "1"
+        RAY_CI_MACOS_WHEELS_AFFECTED = "1"
+
+    print("export RAY_CI_TUNE_AFFECTED=\"{}\"".format(RAY_CI_TUNE_AFFECTED))
```

Comment:
```suggestion
    print("export RAY_CI_TUNE_AFFECTED={}".format(RAY_CI_TUNE_AFFECTED))
```

We don't actually need the quotes I believe. I think it's a bit cleaner without them. Let's remove them (here and below).

Reply:
In .travis.yml, it will still be `"1"`.